lib/execution_engine2/db/MongoUtil.py

```diff
@@ -420,6 +447,26 @@ def update_job_status(self, job_id, status, msg=None, error_message=None):
     def mongo_engine_connection(self):
         yield self.me_connection
 
+    def insert_jobs(self, jobs_to_insert: List[Job]) -> List[ObjectId]:
+        """
+        Insert multiple job records using MongoEngine
+        :param jobs_to_insert: Multiple jobs to insert at once
+        :return: List of job ids from the insertion
+        """
+        # TODO Look at pymongo write_concerns that may be useful
+        # TODO see if pymongo is faster
+
+        inserted = Job.objects.insert(doc_or_docs=jobs_to_insert, load_bulk=False)
+
+        return inserted
+        # TODO: Send Kafka messages here or elsewhere
```

Comment:
I'd recommend not coupling mongoutils and kafka, personally

Reply:
One advantage is that the user doesn't have to remember to create a kafka message, but yea i didn't end up doing that in this PR

Reply:
But the user may not want to sent a kafka message

Reply:
Well the user would be wrong not to do so , because we are supposed to send kafka messages on status transitions.

Reply:
For this particular usage of the module, not necessarily all possible usages of the module. This gets back into separation of concerns and making a module responsible for one thing - in this case interfacing with mongo.